sktime/distances/base/_base.py

```diff
@@ -73,12 +73,12 @@ def distance_factory(
 
         no_python_callable = self._distance_factory(x, y, **kwargs)
 
-        if not hasattr(no_python_callable, "signatures"):
-            raise RuntimeError(
-                "The distance metric specified could not be no_python"
-                "compiled. Try again and if the problem persists raise"
-                "an issue."
-            )
+        # if not hasattr(no_python_callable, "signatures"):
```

Comment:
remove comments. Also, what are we removing here, and why?

Reply:
removed, @chrisholder what was this for?